packages/router/src/grpc/view-protocol-server/tx-planner.ts

```diff
@@ -42,6 +43,7 @@ export const handleTxPlannerReq = async (
   }
 
   if (req.memo) {
+    req.memo.sender ??= refundAddr;
     planner.memo(req.memo);
```

Comment:
if planner.memo is skipped, the transaction can successfully plan and authorize. this agrees with TransactionPlannerRequest, but is that the intended behavior?

if planner.memo is called with a MemoPlaintext that contains a "text" but no "sender", it fails, despite MemoPlaintext.sender being optional.